src/gurt/telemetry.c

```diff
@@ -1123,20 +1123,20 @@ void
 d_tm_print_node(struct d_tm_context *ctx, struct d_tm_node_t *node, int level,
 		char *path, int format, int opt_fields, FILE *stream)
 {
-	struct d_tm_stats_t	stats = {0};
-	struct timespec		tms;
-	uint64_t		val;
-	time_t			clk;
-	char			time_buff[D_TM_TIME_BUFF_LEN];
-	char			*timestamp;
-	char			*name = NULL;
-	char			*desc = NULL;
-	char			*units = NULL;
-	bool			stats_printed = false;
-	bool			show_timestamp = false;
-	bool			show_meta = false;
-	int			i = 0;
-	int			rc;
+	struct d_tm_stats_t stats = {0};
+	struct timespec     tms;
+	uint64_t            val;
+	time_t              clk;
+	char                time_buff[D_TM_TIME_BUFF_LEN];
+	char               *timestamp      = NULL;
```

Comment:
This part of the change is intentional, timestamp is being logged without being set so was causing valgrind warnings, as well as nonsense in the log file.